tools/common/Frameworks.cs

```diff
@@ -650,6 +648,8 @@ public static Frameworks GetMacCatalystFrameworks ()
 
 			// Add frameworks that are not in iOS
 			catalyst_frameworks.Add ("AppKit", 13, 0);
+			catalyst_frameworks.Add ("ScriptingBridge", 13, 0);
+			catalyst_frameworks.Add ("CoreWlan", "CoreWLAN", 15, 0);
```

Comment:
Commit says `ScriptingBridge` any reason for `CoreWLAN` to also come over?

Reply:
This was added in a previous commit. @rolfbjarne just moved it from line 597 to here, probably to put it with the AppKit one.  I'm guessing we had a PR race between me and @rolfbjarne

Reply:
> This was added in a previous commit. @rolfbjarne just moved it from line 597 to here, probably to put it with the AppKit one. I'm guessing we had a PR race between me and @rolfbjarne

Yes, that's exactly what happened.